### ISSUE TITLE
pre-commit support?

### ISSUE THREAD

**Issue by Zeitsperre** [tlocke/flake8-alphabetize#10]
Hi,

This is a neat library that I could see being useful for some of my projects. I'm wondering if you were planning to provide support for running flake8-alphabetize as a [pre-commit hook](https://pre-commit.com/hooks.html). On a whim, I tried adding this to my existing configuration, but it doesn't seem to be working:
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/me/miniconda3/envs/ravenpy/lib/python3.9/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/tme/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/checker.py", line 79, in _mp_run
    return FileChecker(
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/checker.py", line 524, in run_checks
    self.run_ast_checks()
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/checker.py", line 426, in run_ast_checks
    for (line_number, offset, text, _) in runner:
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8_alphabetize/core.py", line 38, in __iter__
    errors = _find_errors(Alphabetize.app_names, self.tree)
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8_alphabetize/core.py", line 250, in _find_errors
    errors.extend(_find_elist_errors(elist_nodes))
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8_alphabetize/core.py", line 227, in _find_elist_errors
    actual_list = [name.id for name in node.elts]
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8_alphabetize/core.py", line 227, in <listcomp>
    actual_list = [name.id for name in node.elts]
AttributeError: 'Attribute' object has no attribute 'id'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/bin/flake8", line 8, in <module>
    sys.exit(main())
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/main/cli.py", line 23, in main
    app.run(argv)
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/main/application.py", line 198, in run
    self._run(argv)
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/main/application.py", line 187, in _run
    self.run_checks()
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/main/application.py", line 103, in run_checks
    self.file_checker_manager.run()
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/checker.py", line 234, in run
    self.run_parallel()
  File "/home/me/.cache/pre-commit/repo68sv11va/py_env-python3/lib/python3.9/site-packages/flake8/checker.py", line 203, in run_parallel
    self.results = list(pool.imap_unordered(_mp_run, self.filenames))
  File "/home/me/miniconda3/envs/ravenpy/lib/python3.9/multiprocessing/pool.py", line 870, in next
    raise value
AttributeError: 'Attribute' object has no attribute 'id'
```

Thanks!

**Comment by tlocke** [comment]
Hi @Zeitsperre, in fact this is a bug, but it should now be fixed in version 0.0.21. So if you'd like to try that new version, let me know how you get on!

**Comment by Zeitsperre** [comment]
Hi @tlocke I just tried this out again and no issues! If you'd like, I can add a small section to the readme on how to configure this for pre-commit.

**Comment by tlocke** [comment]
Glad it's working @Zeitsperre. Thinking about documentation for pre-commit, since Alphabetize is a plugin for Flake8, is separate documentation for Alphabetize necessary? Or to put it another way, once you've got Flake8 working with pre-commit, all the Flake8 plugins should also work?

**Comment by Zeitsperre** [comment]
I was thinking of simply having an example of the config, i.e.:
```yaml
repos:
  - repo: https://github.com/pycqa/flake8
    rev: 6.0.0
    hooks:
    - id: flake8
      additional_dependencies: ['flake8-alphabetize']
```
Pre-commit simply needs to know which plugins to install in its virtual environment.

If this information is unnecessary, I think it's safe to close this issue. Thanks again!

**Comment by tlocke** [comment]
Yes, that sounds like a good idea. If you send a PR I'll add it in.

**Comment by Zeitsperre** [comment]
Closed with #11
